public_html/wp-content/themes/wporg-events-2023/postcss/base/breakpoints.pcss

```diff
@@ -22,7 +22,7 @@
  * wporg-parent-2021
  * @link https://github.com/WordPress/wporg-parent-2021/blob/trunk/source/wp-content/themes/wporg-parent-2021/sass/base/_breakpoints.scss
  */
-@custom-media --small-only (max-width: 559px);
+@custom-media --small-only (max-width: 599px);
```

Comment:
Will this cause any side-effects?

Reply:
I thought it's supposed to be `599px` as it's small-only, relevant to small (`600px`) `559px` should be a typo? or not?


This only side effect is the google map would change its height from `430px` to `246px` `40px` earlier. Do you think that's a proble?
![image](https://github.com/WordPress/wordcamp.org/assets/18050944/3b00dcec-05d2-43f3-adc5-2285ae347c3c)

Reply:
No should be fine. Let's merge.